tests/test_integrity.py

```diff
@@ -83,6 +85,22 @@ def test_references(db):
         print(t)
 
 
+def test_publications(db):
+    # Find unused references in the Sources Table
+    stm = except_(select([db.Publications.c.publication]), select([db.Sources.c.reference]))
+    result = db.session.execute(stm)
+    s = result.scalars().all()
+    assert len(s) == 720, f'found {len(s)} unused references'
```

Comment:
Keep this code -- we might want it for something else in the future!  But it's not doing exactly what we want it to. Instead, work on modifying the above test (`test_references`) to include more tables.